control_toolbox/include/control_toolbox/pid.hpp

```diff
@@ -121,12 +121,11 @@ struct AntiWindupStrategy
         "AntiWindupStrategy 'back_calculation' requires a valid positive tracking time constant "
         "(tracking_time_constant)");
     }
-    if (type != NONE && ((i_min > i_max) || !std::isfinite(i_min) || !std::isfinite(i_max)))
+    if (std::isfinite(i_min) && std::isfinite(i_max) && !(i_min < i_max))
```

Comment:
```suggestion
    if ((std::isfinite(i_min) || std::isfinite(i_max)) && (i_min >= i_max))
```
what happens if anyone sets the parameter i_min = +inf and i_max=0 (any finite number)?

Reply:
@ViktorCVS what do you think about this?